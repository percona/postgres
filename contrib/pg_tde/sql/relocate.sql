-- Support pg_tde already being installed
SET client_min_messages = 'warning';
DROP EXTENSION IF EXISTS pg_tde;

CREATE SCHEMA other;

CREATE EXTENSION pg_tde SCHEMA other;

SELECT other.pg_tde_add_key_provider_file('file-vault', '/tmp/pg_tde_test_keyring.per');

SELECT other.pg_tde_grant_key_viewer_to_role('public');

ALTER EXTENSION pg_tde SET SCHEMA public;

DROP EXTENSION pg_tde;

DROP SCHEMA other;
