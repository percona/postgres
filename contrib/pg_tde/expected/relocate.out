-- Support pg_tde already being installed
SET client_min_messages = 'warning';
DROP EXTENSION IF EXISTS pg_tde;
CREATE SCHEMA other;
CREATE EXTENSION pg_tde SCHEMA other;
SELECT other.pg_tde_add_key_provider_file('file-vault', '/tmp/pg_tde_test_keyring.per');
 pg_tde_add_key_provider_file 
------------------------------
                            1
(1 row)

SELECT other.pg_tde_grant_key_viewer_to_role('public');
 pg_tde_grant_key_viewer_to_role 
---------------------------------
 
(1 row)

ALTER EXTENSION pg_tde SET SCHEMA public;
ERROR:  extension "pg_tde" does not support SET SCHEMA
DROP EXTENSION pg_tde;
DROP SCHEMA other;
